pymc/step_methods/metropolis.py

```diff
@@ -140,22 +140,22 @@ def __init__(
 
         Parameters
         ----------
-        vars: list
+        vars : list
             List of value variables for sampler
-        S: standard deviation or covariance matrix
+        S : standard deviation or covariance matrix
             Some measure of variance to parameterize proposal distribution
-        proposal_dist: function
+        proposal_dist : function
```

Comment:
`callable` is more general than function, and I think applies here. It should also link here automatically: https://docs.python.org/3/glossary.html#term-callable (if it doesn't we can create an alias for it)

Reply:
I have changed this one based on your suggestions.